src/integrationTest/sam.test.ts

```diff
@@ -365,14 +465,18 @@ describe('SAM Integration Tests', async function () {
             })
         })
 
-        async function createZipSamApplication(location: string): Promise<void> {
+        async function createSamApplication(location: string): Promise<void> {
             const initArguments: SamCliInitArgs = {
                 name: samApplicationName,
                 location: location,
-                template: helloWorldTemplate,
-                runtime: scenario.runtime,
                 dependencyManager: getDependencyManager(scenario.runtime),
             }
+            if (scenario.baseImage) {
+                initArguments.baseImage = scenario.baseImage
+            } else {
+                initArguments.runtime = scenario.runtime
```

Comment:
Does this contradict the comment here? https://github.com/aws/aws-toolkit-vscode/pull/1423/files#diff-4b86e3cc81505da2a3b4c8faf1f8ac6d616ae2dec5488e8656d083bc111a1f94R386